cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -287,6 +287,28 @@ def test_video_supported_file_formats(self, files, expected_status, mock_conn, m
             self.assertEqual(response['error'], "Request 'files' entry contain unsupported content_type")
 
     @override_settings(AWS_ACCESS_KEY_ID="test_key_id", AWS_SECRET_ACCESS_KEY="test_secret")
+    @patch("boto.s3.connection.S3Connection")
+    def test_special_character_file(self, mock_conn):
+        """
+        Test that video uploads throws error message when file name contains special characters.
```

Comment:
Also, i find it more readable as `test_upload_with_non_ascii_charaters` than `test_special_character_file`, but I am open :)